client/models/article.coffee

```diff
@@ -31,6 +33,32 @@ module.exports = class Article extends Backbone.Model
   getSlug: ->
     ((@get('author')?.name + '-' + @get('thumbnail_title')?.replace(/[.,\/#!$%\^&\?*;:{}=\-_`~()]/g,'')).toLowerCase()).replace(/\ /g,'-')
 
+  getFullSlug: ->
```

Comment:
Minor, but `getFullSlug` is kinda confusing b/c this is a full url. We've been using the name `href` for this kind of method in Force, but `getArtsyUrl` or something like that could also work.